ui/src/tasks/actions/index.ts

```diff
@@ -296,7 +312,7 @@ export const updateTaskStatus = (task: Task) => async dispatch => {
 
 export const updateTaskName = (task: Task) => async dispatch => {
   try {
-    await client.tasks.update(task.id, task)
+    await api.patchTask({taskID: task.id, data: task})
```

Comment:
These two functions are not equivalent. client.tasks.update has:
```
    if (!!updates.cron) {
      delete original.every
    }

    if (!!updates.every) {
      delete original.cron
    }
```
which we would have to do in the UI code if we are switching from one client to the other

Reply:
Good eyes. I'll make this change happen

Reply:
Great point, let me know if you're around to talk strategy on it